src/solvers/smt2/smt2_tokenizer.h

```diff
@@ -53,6 +53,7 @@ class smt2_tokenizert:public parsert
 
 protected:
   std::string buffer;
+  bool quoted_symbol;
```

Comment:
Missing initialiser. The simplest solution might be to just do it here rather than having to amend any constructors.
```suggestion
  bool quoted_symbol = false;
```

Reply:
done